duckduckgo/duckduckgo-autofill
# Adds a close X button to the top right of the tooltip

**Reviewer:** @shakyShane 
**Asana:** https://app.asana.com/0/0/1204088031977625/f

## Description
- Adds a close X button to the top right of the tooltip
- Updates copy
- Updates pixel naming and replaces initial dismissal pixel with close pixel

![incontext-signup-close](https://user-images.githubusercontent.com/635903/222159775-df88ad77-d96b-45bc-babe-71a9470c0a5c.gif)

## Steps to test
1. Install [extension branch `abrown/changes-to-in-context-signup-treatment`](https://github.com/duckduckgo/duckduckgo-privacy-extension/compare/abrown/changes-to-in-context-signup-treatment?expand=1) with this branch of autofill
2. Install extension and confirm in-context signup shows
3. Use "x" to close the tooltip. Confirm in-context signup can still be re-opened and closed again
